### DISCUSSION TITLE
[spam] Requirement for some fonts, especially Chinese fonts

### DISCUSSION THREAD

**Discussion by qaq311** [gsantner/markor#1679] [Ideas] [answered]
**Font Requirement**
* This problem is an extended requirement for software functionality.
* When using this software, I found that there are few good Chinese fonts. It will be better if it have a wider choice.
Thanks.

**Comment by jayunx** [comment]
This is a good issue, and we’ve also found it. We will provide some font packs. You can try it out.

**Comment by jayunx** [comment]
I have pulled request.

**Comment by qaq311** [comment]
> I have pulled request.

thanks.

**Comment by gsantner** [comment, marked as answer]
what are you guys doing? Some school/university project? I see lots of new people commenting first time today here, and joined GitHub recently. 

> This is a good issue, and we’ve also found it. We will provide some font packs. You can try it out.

As said in other ticket, don't try to impersonate or speak up for the existing Markor developers and contributors. So don't say **you will provide XYZ** at Markor. 

I'm fine if you tell me your plan and if it's in good faith. When you play fair and civil in the project. But please understand we don't want spam or generally malicious intents/behaviour here. Everybody is happy to help students and schoolers to learn.

**Reply by qaq311** [reply]
> what are you guys doing? Some school/university project? I see lots of new people commenting first time today here, and joined GitHub recently.
> 
> > This is a good issue, and we’ve also found it. We will provide some font packs. You can try it out.
> 
> As said in other ticket, don't try to impersonate or speak up for the existing Markor developers and contributors. So don't say **you will provide XYZ** at Markor.
> 
> I'm fine if you tell me your plan and if it's in good faith. When you play fair and civil in the project. But please understand we don't want spam or generally malicious intents/behaviour here.

I am a student taking a class call software engineering. tomorrow is deadline. the project is to fix issue in github project and we choose this project. but our level is low and not have android development experience. We beg you donot close our new issue. we are so sorry but we have no choice. we have no malice.
sorry again
thanks.

**Reply by gsantner** [reply]
I understand. But please tell your colleagues to note what is the matter when you create such issues/Pull requests. If it can be ignored, and you just do it for testing - this is OK. I just won't merge anything of your experiments.

It is often not clear what is spam, good to know in advance you do that for educational purpose.

Being able to work together, respecting the rules and communication is just like coding itself one of the core aspects of working on (open source) software projects.

Thanks

**Reply by qaq311** [reply]
Thank you very much for your understanding. We are very sorry for this behavior. We just want to do some homework required by our school, create such issues/Pull requests. You don't need to care about these tests at all. What's more, could you please not delete these issues for the time being?

In addition, we need some screenshots of issues and Pull requests, can you delete the previous comments? Thank you again for your understanding and I am sorry for this behavior.

**Reply by qaq311** [reply]
> please just create new `whatever` you guys need.

We want to create something new, but we're not there yet. However, we continue to learn and hope to make a real contribution in the future. Once again, I'm sorry for my behavior.

**Reply by gsantner** [reply]
I mean new comment, new pull request, new issue :smile: .

You don't need to apologize more, I know now. Just if you do such thing next time (whereever) tell the people in advance. Everybody is happy to help students and schoolers to learn.

**Reply by qaq311** [reply]
> I mean new comment, new pull request, new issue 😄 .
> 
> You don't need to apologize more, I know now. Just if you do such thing next time (whereever) tell the people in advance. Everybody is happy to help students and schoolers to learn.

Since we need issues and Pull requests for our homework this time, could you please temporarily keep these issues?

**Reply by gsantner** [reply]
@qaq311 Are you all done with your homework and can I close?

**Reply by Shallow-D12** [reply]
> @qaq311 Are you all done with your homework and can I close?

Yes, thank you very much.
